src/server/project.ts

```diff
@@ -130,6 +130,7 @@ namespace ts.server {
         protected languageService: LanguageService;
 
         public languageServiceEnabled = true;
+        public exceededFilename: string | undefined;

```

Comment:
Please make this internal

Reply:
Making it private, as is not used outside from this class.